mne/decoding/search_light.py

```diff
@@ -8,9 +8,10 @@
 from .base import BaseEstimator, _check_estimator
 from ..parallel import parallel_func
 from ..utils import (_validate_type, array_split_idx, ProgressBar,
-                     verbose)
+                     verbose, fill_doc)
 
 
+@fill_doc
```

Comment:
why is this decorator needed on classes and nothing is needed in functions?

Reply:
Because `verbose` has been augmented to be what it was before, plus a doc filler. (If it hadn't been, then we would need `@fill_doc\n@verbose` everywhere). Our classes are not decorated, so they need a doc-filling decorator.